### PR TITLE
Update SLR2.md

### PR DESCRIPTION
**Pull request by russdan**
Updated with pairing instructions, corrected message endpoints (heat/water are part of the endpoint not the message), added emergency_heating state and added command sequences to switch water on/boost/off.

### CODE REVIEW

File: docs/devices/SLR2.md [hunk 1]
```diff
@@ -28,17 +39,33 @@ description: "Integrate your Hive SLR2 via Zigbee2MQTT with whatever smart home
...
+- `temperature_setpoint_hold_duration`: Temperature setpoint hold duration (emergency_heating only) - used to determine how long the emergency heating period isrelay. To control publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/water/set` with payload `{temperature_setpoint_hold_duration": VALUE}` where `VALUE` is number of minutes. There is no read command for this value (i.e. write-only).
+- `system_mode`: Mode of this device. To control publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/water/set` with payload `{"system_mode": VALUE}` where `VALUE` is one of: `off`, `auto`, `heat`,`emergency_heating` . To read send a message to `zigbee2mqtt/FRIENDLY_NAME/water/get` with payload `{"system_mode": ""}`.
+- `running_state`: The current running state. Read-only. Possible values are: `idle`, `heat`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/water/get` with payload `{"running_state": ""}`.
+
+To switch water ON send the following messages / payloads:
```
Comment (Koenkk) [inline docs/devices/SLR2.md:54]: Please don't make manual changes in this part of the documentation, they are auto generated based on the `exposes`. You can move it under the pairing section.

File: docs/devices/SLR2.md [hunk 2]
```diff
@@ -28,17 +39,33 @@ description: "Integrate your Hive SLR2 via Zigbee2MQTT with whatever smart home
...
-- `local_temperature`: Current temperature measured on the device (in °C). To read send a message to `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"local_temperature_heat": ""}`.
-- `system_mode`: Mode of this device. To control publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set` with payload `{"system_mode_heat": VALUE}` where `VALUE` is one of: `off`, `auto`, `heat`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"system_mode_heat": ""}`.
-- `running_state`: The current running state. Possible values are: `idle`, `heat`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"running_state_heat": ""}`.
+- `occupied_heating_setpoint`: Temperature setpoint. To control publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/heat/set` with payload `{"occupied_heating_setpoint": VALUE}` where `VALUE` is the °C between `7` and `30`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/heat/get` with payload `{"occupied_heating_setpoint": ""}`.
```
Comment (Koenkk) [inline docs/devices/SLR2.md:42]: Why has this been changed?

Comment (russdan) [reply docs/devices/SLR2.md:42]: Hi, the same reason as for water in that the auto-generated endpoint names are incorrect - the endpoint is `zigbee2mqtt/FRIENDLY_NAME/heat/set` for example, with a payload of `{"occupied_heating_setpoint": VALUE}`, not `zigbee2mqtt/FRIENDLY_NAME/set` for example, with a payload of `{"occupied_heating_setpoint_heat": VALUE}` as per auto- generated values.

File: docs/devices/SLR2.md (hunk 1 above)
Comment (russdan) [reply docs/devices/SLR2.md:54]: also the auto-generated values for `system_mode` are missing `emergency_heating`  for example

File: docs/devices/SLR2.md (hunk 2 above)
Comment (Koenkk) [reply docs/devices/SLR2.md:42]: `occupied_heating_setpoint_heat` should also work, but there was a bug which has been fixed recently. This fix is only available in the dev branch  (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html).  (https://github.com/Koenkk/zigbee2mqtt/issues/7470)

Comment (russdan) [reply docs/devices/SLR2.md:42]: Hi, the fix to `occupied_heating_setpoint_heat` was the one I spotted and fixed :)    

I hadn't appreciated that both methods work, but I've confirmed that now, ie `zigbee2mqtt/FRIENDLY_NAME/heat/set` with a payload of `{"occupied_heating_setpoint": VALUE}` and `zigbee2mqtt/FRIENDLY_NAME/set` with a payload of `{"occupied_heating_setpoint_heat": VALUE}` both work - sorry for the confusion!  

I'll rework the .md as you suggest.

Comment (russdan) [reply docs/devices/SLR2.md:42]: Hi @Koenkk can you please "request changes" to the PR and I'll update?  I can't edit it now its in review state... thanks :)

Comment (Koenkk) [reply docs/devices/SLR2.md:42]: @russdan I don't understand why you cannot edit this, you should be able to push to your [own branch](https://github.com/russdan/zigbee2mqtt.io/tree/patch-1) and this pr will get updated

**Pull request closed without merging**